catalogue_api/api/src/test/scala/uk/ac/wellcome/platform/api/services/WorksServiceTest.scala

```diff
@@ -3,15 +3,15 @@ package uk.ac.wellcome.platform.api.services
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.{FunSpec, Matchers}
 import uk.ac.wellcome.models.{IdentifierSchemes, SourceIdentifier}
-import uk.ac.wellcome.platform.api.WorksUtil
 import uk.ac.wellcome.platform.api.fixtures.{
   ElasticsearchServiceFixture,
   WorksServiceFixture
 }
 import uk.ac.wellcome.display.models.{
   DisplayIdentifier,
   DisplayWork,
-  WorksIncludes
+  WorksIncludes,
+  WorksUtil
```

Comment:
Is it worth putting this in a test namespace?

Reply:
Maybe, but I don’t care enough right now to wait another CI cycle. 🙃